rados/errors.go

```diff
@@ -48,8 +48,15 @@ func getErrorIfNegative(ret C.int) error {
 // Public go errors:
 
 var (
-	// ErrNotConnected is returned when functions are called without a RADOS connection
+	// ErrNotConnected is returned when functions are called without
```

Comment:
Minor nit: I much prefer to see changes that only reformat things use a separate commit from changes that add or modify behavior.